bedrock/base/templates/includes/site-footer.html

```diff
@@ -15,7 +15,7 @@ <h1 class="logo"><a href="{{ url('mozorg.home') }}">Mozilla</a></h1>
               <li><a href="{{ url('mozorg.contact.spaces.spaces-landing') }}">{{ _('Contact Us') }}</a></li>
               <li class="wrap"><a href="{{ url('mozorg.partnerships') }}">{{ _('Partner with Us' ) }}</a></li>
               <li class="clear"><a href="{{ donate_url('mozillaorg_footer') }}" class="donate">{{ _('Donate') }}</a></li>
-              <li class="wrap"><a href="https://affiliates.mozilla.org/">{{ _('Firefox Affiliates') }}</a></li>
+              <li class="wrap"><a href="{{ url('mozorg.contribute.friends') }}">{{ _('Firefox Friends') }}</a></li>
```

Comment:
I asked to engagement for the landing page, it's not supposed to be localizable. Leave the wrapping, I simply won't expose the string.